lib/services/identity_service.dart

```diff
@@ -32,15 +31,15 @@ class IdentityService extends ChangeNotifier with LogMixin {
 
   Future setName(String name) async {
     final updatedName = await _repository.put(_NAME_KEY, name);
-    notifyListeners();
+    if (changeCallback != null) changeCallback!();
```

Comment:
Is there a reason you used this line here instead of `invokeChangeCallback()`?

Reply:
There is not and I changed it.